test/unit/karma.bulkImport.conf.js

```diff
@@ -4,18 +4,28 @@ module.exports = function (config) {
     'use strict';
     const karmaConfig = require('./karma.conf');
     karmaConfig(config);
-    const bulkImportTestPath = 'test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js';
+    const bulkImportTestPaths = [
+        'test/unit/spec/nbextensions/bulkImportCell/main-spec.js',
+        'test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js',
```

Comment:
Modified this file to use the other `bulkImportCell` core tester. Eventually, I'll get to having it all return as a single resolved Promise on startup, really I will...

Reply:
I think the bulkImportCell-spec tests are OK now, but there's still at least one other module that's causing unresolved promise issues. I wonder if we should make the naming of this section more generic so that any potentially dodgy test files can be put into this "sin bin" and executed in a separate run.

Reply:
Good idea. I'll genericise the naming.

Reply:
Alrighty, changed that to "karma.isolated.conf.js" to show that they should be isolated.

Although, "karma.sin-bin.conf.js" was inviting.